docker/Dockerfile

```diff
@@ -52,7 +52,7 @@ RUN install_remote \
         mrc-ide/eppasm \
         mrc-ide/heartbeatr \
         mrc-ide/rrq \
-        reside-ic/traduire
+        reside-ic/traduire@master
```

Comment:
This is also present in #130 and was needed to force TeamCity to invalidate the build cache, but it is harmless